pyatv/const.py

```diff
@@ -5,7 +5,7 @@
 
 MAJOR_VERSION = "0"
 MINOR_VERSION = "10"
-PATCH_VERSION = "3"
+PATCH_VERSION = "4"
```

Comment:
Same here, you should not update version manually.

Reply:
Sorry!  think I had to here - to double check had pulled fork version.
But feel free to please cancel and update you end (away for a bit)